src/libs/actions/PersonalDetails.js

```diff
@@ -310,6 +310,14 @@ function openPersonalDetailsPage() {
     API.read('OpenPersonalDetailsPage');
 }
 
+/**
+ * Fetches public profile info about a given user
```

Comment:
Do we want to add a note that the logged in user will see their details (like pronouns, timezone, email) only if we've loaded that info in other places in the app?

Reply:
Good call. Made the comment clearer.